lib/aurora_bootstrapper/utility.rb

```diff
@@ -0,0 +1,23 @@
+module AuroraBootstrapper
+  class Utility
+    class << self
+      def db_pass
+        ENV.fetch( 'DB_PASS' ) do
+          File.open( ENV.fetch( 'DB_CRED_FILE' ) ).read
+        end
+      end
+
+      def db_user
+        ENV.fetch( 'DB_USER' ) do
+          File.open( ENV.fetch( 'DB_USER_FILE' ) ).read
+        end
+      end
+
+      def rollbar_token
+        ENV.fetch( 'ROLLBAR_TOKEN' ) do
+          File.open( ENV.fetch( 'ROLLBAR_TOKEN_FILE' ) ).read
+        end
+      end
```

Comment:
do you think it would be over the top to have a shared function to do the env/file fetching like
```ruby
class Utility
  class << self
    def db_pass
      from_env 'DB_PASS'
    end
    
    def db_user
      from_env 'DB_USER'
    end
  
    def rollbar_token
      from_env 'ROLLBAR_TOKEN'
    end
    
    private
    
    def from_env( var )
      ENV.fetch( var ) do
        File.open( ENV.fetch( "${var}_FILE" ) ).read
      end
    end
  end
end
```

and then you could - if you wanted to get super meta - combine them using [method_missing](https://riptutorial.com/ruby/topic/1076/method-missing), like
```ruby
class Utility
  class << self 
    def method_missing( m, *args, &block )
      if ['db_pass', 'db_user', 'rollbar_token'].include? m
        from_env m.upcase
      else
        super m, *args, &block
      end
    end

    private
    
    def from_env( var )
      ENV.fetch( var ) do
        File.open( ENV.fetch( "${var}_FILE" ) ).read
      end
    end
  end
end
```

just options. you don't have to. : D

Reply:
Nice!

Reply:
Ruby master :D

Reply:
I will use the first option